nbgrader/nbextensions/create_assignment/create_assignment.css

```diff
@@ -35,6 +35,15 @@ div.nbgrader-highlight div.text_cell_render, div.nbgrader-highlight div.input_ar
     border: 1px solid #337ab7 !important;
 }
 
+
+.celltoolbar {
+    // safari fix, we cannot use -webkit-flex on hbox
+    // and vbox either or all css get borked
+    // cf https://github.com/jupyter/nbgrader/issues/394.
+    // manual workaround while #370 is not released. 
+    display: -webkit-flex;
+};
```

Comment:
Ok, got it: you're also using javascript comments rather than CSS, I think, so safari is breaking because of the comment syntax. If it's fixed to `/*...*/` then it seems to work in safari too :smile: